From 7b793ece1e8452ab41ad2287dd206feb8327e0ad Mon Sep 17 00:00:00 2001
From: zain wang <wzz@rock-chips.com>
Date: Fri, 19 May 2017 14:00:46 +0800
Subject: [PATCH] mfd: fusb302: fix some variable following PD specification

In the spec:
The tTypeCSendSourceCap is defined from 100ms to 200ms.
The nCapsCount is defined 50.

Change-Id: I09bcdb7a83c353ab099d51228cf8ca13e562d839
Signed-off-by: zain wang <wzz@rock-chips.com>
---
 drivers/mfd/fusb302.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/mfd/fusb302.h b/drivers/mfd/fusb302.h
index 905a0c8ef33b..7a757deb8f49 100644
--- a/drivers/mfd/fusb302.h
+++ b/drivers/mfd/fusb302.h
@@ -279,12 +279,12 @@ enum tcpm_rp_value {
 #define VDM_TYPE_BUSY		3
 
 #define N_DEBOUNCE_CNT		(10 - 1)
-#define N_CAPS_COUNT		100
+#define N_CAPS_COUNT		50
 #define N_HARDRESET_COUNT	0
 
 #define T_NO_RESPONSE		5000
 #define T_SRC_RECOVER		830
-#define T_TYPEC_SEND_SOURCECAP	3
+#define T_TYPEC_SEND_SOURCECAP	100
 #define T_SENDER_RESPONSE	30
 #define T_SRC_TRANSITION	30
 #define T_TYPEC_SINK_WAIT_CAP	500
-- 
2.35.3

